docs/installation.rst

```diff
@@ -123,38 +123,75 @@ For example, on Debian-based Linux we need to also do:
 
 .. code::
 
-        sudo apt-get install libnetcdf-mpi-dev 
-        sudo apt-get install openmpi-bin libopenmpi-dev 
+        sudo apt-get install libnetcdf-mpi-dev
+        sudo apt-get install openmpi-bin libopenmpi-dev
 
 The cmake call has to enable MPI support:
 
 .. code::
 
-        cmake .. -DENABLE_MPI=ON 
+        cmake .. -DENABLE_MPI=ON
 
 You might need to tell cmake which compiler to use, e.g.
 
 .. code::
 
-        cmake .. -DCMAKE_CXX_COMPILER=/usr/bin/mpicxx -DENABLE_MPI=ON 
+        cmake .. -DCMAKE_CXX_COMPILER=/usr/bin/mpicxx -DENABLE_MPI=ON
 
-Using Dockerfiles for Dependencies and Building
------------------------------------------------
+Using Dockerfiles for Development or Production Runs
+----------------------------------------------------
 
-In the Dockerfiles directory we provide two versions of Dockerfiles: with and without MPI support. You need to have _Docker installed on your computer. Then you can build and run it from the root of your NextSimDG repository, for example, with 
+In the ``Dockerfiles`` directory we provide two ``Dockerfile``'s are provided.
 
-.. code::
+- ``Dockerfile.devenv`` - This is the ``Dockerfile`` used to build the development image (``ghcr.io/nextsimhub/nextsimdg-dev-env``), that is
+  used in the GitHub CI.
+- ``Dockerfile.production`` - This ``Dockerfile`` is based off of the development image and it
+  additionally installs ``nextsim`` so that you can run on any machine with ``docker`` installed.
+
+Development Dockerfile
+^^^^^^^^^^^^^^^^^^^^^^
+
+A development image is provided on the nextsimhub `GitHub container registry
+<https://github.com/orgs/nextsimhub/packages>`_ because it is needed for the CI.
+
+If in future, this needs to be replaced. Please see `instructions
+<https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry>`_
+on the GitHub website.
+
+To build the docker image, please use these instructions:
+
+.. code-block:: console
+
+    docker build --file Dockerfile.devenv . -t ghcr.io/nextsimhub/nextsimdg-dev-env:latest
+
+.. note::
+   The formatting of the image name **is important**. The format is
+   ghcr.io/NAME_OF_REPOSITORY/NAME_OF_IMAGE:TAG
+
+If you want to test or use the image locally, use the following command:
+
+.. code-block:: console
+
+   docker pull ghcr.io/nextsimhub/nextsimdg-dev-env:latest
+
+Production Dockerfile
+^^^^^^^^^^^^^^^^^^^^^
+
+The production image is not stored on the nextsimhub `GitHub container registry
+<https://github.com/orgs/nextsimhub/packages>`_ because it is not needed for the CI. Users of the
```

Comment:
But might it be worth testing the production image every once in a while? Otherwise it could break without us noticing. To reduce doing this unnecessarily often, it could be done using a scheduled GitHub Actions job, perhaps once a week or month.

[I might be overthinking this as I see that the production image is based off the development environment and doesn't change much other than configuration options. Thought it would be worth mentioning, though.]

Reply:
Yeah, we do not want to support this at present.

The idea of production release is more for users that want a simpler installation of nextsim for testing. The name is perhaps (currently) misleading because I would not recommend using it on HPC because it will not use the system MPI (amongst other core deps) and this could affect performance. But it might be useful for other scientists who want to try/test new features e.g., XIOS and MPI.